src/Components/Publishing/Layouts/StandardLayout.tsx

```diff
@@ -1,6 +1,9 @@
 import { color, space } from "@artsy/palette"
+import * as Schema from "Analytics/Schema"
+import { getEditorialHref } from "Components/Publishing/Constants"
 import { get, omit } from "lodash"
 import React from "react"
+import track from "react-tracking"
```

Comment:
Did you on purpose import this one instead of `track` from `"Analytics"` ?

Reply:
Oh cool- didn’t realize that was a thing, had remembered you said to use library rather than removed utils version, but sounds like that was only for untyped calls.  Will update!

Reply:
Yeah sorry about that, I should have written that up for you!